### DISCUSSION TITLE
Cannot submit for review - A review submission is already in progress

### DISCUSSION THREAD

**Discussion by valentin-at-edlio** [fastlane/fastlane#20961] [Q&A]
When I try to submit a new build for an app that have been rejected I run into this error:
`Cannot submit for review - A review submission is already in progress`

This happens because the previous app submission is still in progress on App Store Connect and when you go to the details it even says:
_"Your app version was rejected and no other items submitted can be accepted or approved. You can make edits to your app version below."_

Is there any way to cancel/resubmit a submission with fastlane?

**Comment by timothycosta** [comment]
I would like to know if this is possible also. We have 15 apps. When apple reviews all of them together, they will often reject multiple at once for the same issue.
